python/python/res/server/ertrpcserver.py

```diff
@@ -39,24 +40,14 @@ def createFault(error, message):
     error_code = INVERSE_FAULT_CODES[error]
     return Fault(error_code, message)
 
-
-
 class ErtRPCServer(SimpleXMLRPCServer):
-    def __init__(self, config, host="localhost", port=0, log_requests=False, verbose_queue=False):
+    def __init__(self, enkf_main=None, res_config=None, host="localhost", port=0, log_requests=False, verbose_queue=False):
```

Comment:
Sure thing. I've deprecated providing a configuration file and instead given the possibility of giving a res_config. I also kept the old possibility of providing an enkf_main (which seems to be the approach used in ERT at least..). Which of the options do we prefer?